### ISSUE TITLE
There's something breaking my site with live category

### ISSUE THREAD

**Issue by jengajenga** [WWBN/AVideo#5945]
TL:DR; I have issues with the categories, notification bar and live front page display.

==

I can confirm my nginx is on - all streamers are visible in the stat page.

However I cannot display the same on the front page. It shows me 2 in the notification bar, and those two are HLS links from elsewhere. Then it pops back and shows the correct number, then it goes back to two. If it was not for the HLS links i added, it would be zero.

In my setting: doNotshowLiveonCategoryList is NOT selected so it is suppose to show..

I have cleared front page and directory cache but still nothing. my front page is blank

![live-tv](https://user-images.githubusercontent.com/65451390/140876058-b55b24f2-1c80-409a-8de8-67bdd1320962.jpeg)

I have no user groups created

I've done the latest git - no change

I've done a hard master/origin reset, no change.

How it should be:
![live-correct](https://user-images.githubusercontent.com/65451390/140879221-1faa452f-bb4b-490f-b65e-640da3cc4623.jpeg)

How it is:
![live-wrong](https://user-images.githubusercontent.com/65451390/140879333-d37d2c50-ee9b-4456-b245-a566378d6f6a.jpeg)

See this error: Showing 2 in the bar but showing all the other lives when you click the drop down. First page is blank.
![live-notification](https://user-images.githubusercontent.com/65451390/140879374-56f986a3-d4ef-45c5-b484-6a743d503cb2.PNG)

Categories are 4. Two are missing from front page where they should be
![category-error](https://user-images.githubusercontent.com/65451390/140879674-8332d91b-d184-4138-b58f-a2bf69f2844a.jpeg)

Please assist

Thank you

**Comment by jengajenga** [comment]
Socket is on and working very well

**Comment by jengajenga** [comment]
When I enable ShowLiveOnVideosList, then it populates together with the categories.

I don't want to show the normal live.

I want to show the categories live because i have different live categories

But I don't think that is the real issue.

When I create categories, they appear in the database. but they don't appear on the site until after some hours. sometime even after a day.

**Comment by DanielnetoDotCom** [comment]
when you create a new category please clear your cache.

I am not sure exactly what is your issue. can you put a web page with your issue?

**Comment by jengajenga** [comment]
> when you create a new category please clear your cache.
> 
> I am not sure exactly what is your issue. can you put a web page with your issue?

you will have to log in and see. right now i have disabled the LIVE list via mainArea.php file so it is showing Categories. but if I remove LIVE list via Live Settings, category settings also disappear and i have a blank page.

i'll email you the domain.

**Comment by DanielnetoDotCom** [comment]
I see, you want to use livelinks but not use the live correct?

**Comment by JoshWho** [comment]
I think he wants to turn the top bell off and still keep the live streams visible on the main page.

As of right now if you Turn Off Notification in the Live Plugin settings no more streams show on the home page.

**Comment by jengajenga** [comment]
thank you @DanielnetoDotCom and @JoshWho for giving it a shot. let me put it here

**Comment by jengajenga** [comment]
Closed by mistake

**Comment by DanielnetoDotCom** [comment]
If you clear the cache all shows fine?

do you think it is a cache but or do you think it is just a configuration?
